ee/libcglue/samples/mem_test/main.c

```diff
@@ -52,7 +53,7 @@ void _libcglue_rtc_update() {}
 // "weak" function called by crt0.o
 void _ps2sdk_memory_init()
 {
-    if (GetMemorySize() == (32 * 1024 * 1024) && IsDESRMachine() == 1) {
+    if (GetMemorySize() == (32 * 1024 * 1024) && GetSyscallHandler(__NR_GetMemoryMode) && GetMemoryMode() == 1) {
```

Comment:
should we create a macro to make easier the usage of this `GetSyscallHandler(__NR_GetMemoryMode) && GetMemoryMode() == 1`? or even the whole `if` condition

Reply:
🆗